cli/Makefile.toml

```diff
@@ -51,3 +53,42 @@ git push --tags
 script = '''
 npx prettier --write .
 '''
+
+[tasks.test]
+clear = true
+command = "cargo"
+args = ["nextest", "run"]
+
+[tasks.clippy]
+clear = true
+command = "cargo"
+args = ["clippy", "--locked", "--tests", "--", "-D", "warnings"]
+
+[tasks.clippy-nightly]
+toolchain = "nightly"
+command = "cargo"
+args = ["clippy", "--locked", "--tests", "--", "-D", "warnings"]
+
+[tasks.run]
+clear = true
+command = "cargo"
+args = ["run", "--", "${@}"]
+
+[tasks.run-release]
+clear = true
+command = "cargo"
+args = ["run", "--release", "--", "${@}"]
+
+[tasks.update-workers]
+script = '''
+cd ../../api/workers/gateway
+../../scripts/dev/build-cli-assets.sh
```

Comment:
if this is for development consider adding `RELEASE_FLAG="--dev"` and deleting `~/.grafbase`

Reply:
The script already should delete `~/.grafbase`, we could add the dev flag

Reply:
Right, it does the deletion, I've just checked.